client/src/legacy/LeftAndMain.js

```diff
@@ -55,7 +55,7 @@ window.ss.tabStateUrl = function() {
   return window.location.href
     .replace(/\?.*/, '')
     .replace(/#.*/, '')
-    .replace($('base').attr('href'), '');
+    .replace(new RegExp(`^${$('base').attr('href')}/?`), '');
```

Comment:
Should regex escape the base e.g. `.` should become `\.`

Or just `ltrim('/')` the base and then prefix it with `'/'` (better imo)

Reply:
There's no `ltrim` or `rtrim` in javascript. I don't see any downsides to using regex for this. I'll escape the string appropriately.

Reply:
Done